src/controllers/cohort.js

```diff
@@ -11,3 +11,13 @@ export const create = async (req, res) => {
     throw e
   }
 }
+
+export const get = async (req, res) => {
```

Comment:
There is a better name for this function. As its getting _all_ of the cohorts. Likes call it `getAll` or `getAllCohorts`